explorer/frontend/components/FieldPrice.jsx

```diff
@@ -0,0 +1,23 @@
+import styles from '@/styles/components/FieldPrice.module.scss';
+
+const FieldPrice = ({ value, change }) => {
+	let changeText;
+	let changeClassName;
+
+	if (change < 0) {
+		changeText = `${change}%`;
+		changeClassName = styles.changeDecrease;
+	} else {
+		changeText = `+${change}%`;
```

Comment:
you can cover both branches with something like 
``const changeText = `${Math.sign(change)}${change}%` ``

Reply:
@gimre-xymcity actually `Math.sign()` returns `-1`, `0` or `1`, so I don't think it could be used here.